File: java/Processor.java
class Processor{
	// atribut private
    private String name;
	private String price;

	// constructor
	public Processor(){}
	public Processor(String name, String price){
		this.name = name;
		this.price = price;
	}

	// setter/getter
	public void setname(String name) {
		this.name = name;
	}
	public String getname() {
		return name;
	}
	public void setprice(String price) {
		this.price = price;
	}
	public String getprice() {
		return price;
	}

	/*Method*/
	public void printProcessor(){
		System.out.println("Processor name : " + this.name);
		System.out.println("Processor price : " + this.price);
	}
}

File: python/Processor.py
class Processor:
	# atribut private
	__name = ""
	__price = ""

	# constructor
	def __init__(self, name, price):
		self.__name = name
		self.__price = price

	# getter/setter
	def setname(self, name):
		self.__name = name

	def getname(self):
		return self.__name

	def setprice(self, price):
		self.__price = price

	def getprice(self):
		return self.__price

	# method
	def printProcessor(self):
		print("Processor name : " + str(self.__name))
		print("Processor price : " + str(self.__price))
	